File: C++/BubbleSort.cpp
//Bubble Sort,
#include<iostream>
using namespace std;

//Bubble Sort,
void BubbleSort(int DataStore[], int len){
  int temp;
  for(int i=0; i<len; i++){
    for(int j=0; j<len-i-1; j++){
        if(DataStore[j]>DataStore[j+1]){
            temp = DataStore[j];
            DataStore[j]=DataStore[j+1];
            DataStore[j+1] = temp;
        }
    }
  }
}

//Print DataStore,
void PrintData(int DataStore[],int len){
    cout<<"Show Data Store : ";
    for(int a=0; a<len; a++){
        cout<<DataStore[a]<<" ";
    }
    cout<<endl;
}

int main(){
    int DataStore[] = {-2,45,0,11,-9};
    int len = sizeof(DataStore)/sizeof(DataStore[0]);

    BubbleSort(DataStore,len);
    PrintData(DataStore,len);



    return 0;
}

File: Python/BubbleSort.py
# Bubble Sort,

def BubbleSort(DataStore,dataSpace):
    temp =0
    for a in range(0,dataSpace):
        for b in range(0,dataSpace-a-1):
            if(DataStore[b]>DataStore[b+1]):
                temp = DataStore[b]
                DataStore[b] = DataStore[b+1]
                DataStore[b+1] = temp

                



DataStore = [-2,45,0,11,-9]
dataSpace = len(DataStore)
BubbleSort(DataStore, dataSpace)
print("Bubble Sorting DataStore : ",DataStore)